analysis/task_analysis_volume/final_volume_plots.ipynb

```diff
@@ -0,0 +1,217 @@
+    "\n",
+    "    w,h=axial_HC_baseline.size\n",
+    "    area_no_cmap = (0, 20, w-83, h-25)\n",
+    "\n",
+    "    HCb = axial_HC_baseline.crop(area_no_cmap) #crop just the part of the image that you want\n",
+    "\n",
+    "\n",
+    "    axial_MM_baseline_path = glob.glob(f'../../../derivatives/task_analysis_volume/visualization/raw_figures/group-MM_ses-baseline_task-{task}_contrast-{contrast}_threshold-*_n-*_display-axial.png')[0]\n",
+    "    axial_MM_baseline=Image.open(axial_MM_baseline_path,mode='r')\n",
+    "    MMb = axial_MM_baseline.crop(area_no_cmap) #crop just the part of the image that you want\n",
+    "\n",
+    "\n",
+    "    axial_MM_1year_path = glob.glob(f'../../../derivatives/task_analysis_volume/visualization/raw_figures/group-MM_ses-1year_task-{task}_contrast-{contrast}_threshold-*_n-*_display-axial.png')[0]\n",
+    "    axial_MM_1year=Image.open(axial_MM_1year_path,mode='r')\n",
+    "\n",
+    "    area_cmap = (0, 20, w, h-25)\n",
+    "\n",
+    "    MM1 = axial_MM_1year.crop(area_cmap) #crop just the part of the image that you want\n",
+    "\n",
+    "\n",
+    "    w_no_cmap,h_no_cmap = HCb.size\n",
+    "    w_cmap,h_cmap = MM1.size\n",
+    "\n",
+    "    new_im = Image.new('RGB', ( (w_no_cmap*2+w_cmap) , h_no_cmap) ,(255, 255, 255, 1)) #make a new image\n",
+    "    #paste in the cut out images\n",
+    "    new_im.paste(HCb,(0,0)) \n",
+    "    new_im.paste(MMb,(w_no_cmap,0))\n",
+    "    new_im.paste(MM1,(w_no_cmap*2,0))\n",
+    "    \n",
+    "    #create paths to output dir if not exist\n",
+    "    derivatives_path = '../../../derivatives'\n",
+    "    nilearn_output_path = os.path.join(derivatives_path, 'task_analysis_volume','visualization','final_nifti_figures')\n",
+    "    if not os.path.isdir(nilearn_output_path):\n",
+    "        os.makedirs (nilearn_output_path)\n",
+    "\n",
+    "    HCb_threshold = axial_HC_baseline_path.split('threshold-')[1].split('_')[0]\n",
+    "    HCb_part_count = axial_HC_baseline_path.split('_n-')[1].split('_')[0]\n",
+    "\n",
+    "    MMb_threshold = axial_MM_baseline_path.split('threshold-')[1].split('_')[0]\n",
+    "    MMb_part_count = axial_MM_baseline_path.split('_n-')[1].split('_')[0]\n",
+    "\n",
+    "    MM1_threshold = axial_MM_1year_path.split('threshold-')[1].split('_')[0]\n",
+    "    MM1_part_count = axial_MM_1year_path.split('_n-')[1].split('_')[0]\n",
+    "\n",
+    "    new_im.save(f'../../../derivatives/task_analysis_volume/visualization/final_nifti_figures/task-{task}_contrast-{contrast}_HCb_threshold-{HCb_threshold}_HCb_n-{HCb_part_count}_MMb_threshold-{MMb_threshold}_MMb_n-{MMb_part_count}_MM1_threshold-{MM1_threshold}_MM1_n-{MM1_part_count}_nifti_figure.png')\n",
+    "\n",
+    "    return w_no_cmap\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "614849be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_vol_figure_final(task,contrasts,w_no_cmap):\n",
```

Comment:
for the first paper i was involved with as a grad student, we created final figure and then went on to produce revisions of that final figure up to 8 times i think. since then i don't call anything final.

Reply:
renamed for surf and vol!